### ISSUE TITLE
Potential Papers for June 3

### ISSUE THREAD

**Issue by makagan** [drphilmarshall/BreakfastClub#6]
Paper on geometry and metric for comparing distributions (with a focus on GANS): https://arxiv.org/abs/1712.07822
- We could probably find a paper on distribution comparisons more generally if we want.

For further Likelihood-Free Inference work, there is:

Likelihood-free MCMC with Amortized Approximate Ratio Estimators: https://arxiv.org/abs/1903.04057
- Uses likelihood ratio estimation with classifiers to guide MCMC

On Contrastive Learning for Likelihood-free Inference: https://arxiv.org/abs/2002.03712
- Connects previous LFI MCMC paper with APT

**Comment by makagan** [comment]
Hi All, are we meeting tomorrow?

**Comment by jiwoncpark** [comment]
Yes, we are! I do want to get to the GAN distributions soon (we'll probably divide it up over ~2 meetings), but thought we could read the contrastive learning paper for tomorrow -- was there preference for any of the papers (I remember I had to leave early)?
